src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3191,14 +3191,89 @@
 short-summary: Manage private endpoints.
 """
 
+helps['network private-endpoint create'] = """
+type: command
+short-summary: Create a private endpoint.
+examples:
+  - name: Create a private endpoint.
+    text: az network private-endpoint create -g MyResourceGroup -n MyPE --vnet-name MyVnetName --subnet MySubnet --private-connection-resource-id MyPLSId --connection-name tttt -l centralus
+"""
+
+helps['network private-endpoint delete'] = """
+type: command
+short-summary: Delete a private endpoint.
+"""
+
 helps['network private-endpoint list'] = """
 type: command
 short-summary: List private endpoints.
 """
 
 helps['network private-endpoint show'] = """
 type: command
-short-summary: Get the details of an private endpoint.
+short-summary: Get the details of a private endpoint.
+"""
+
+helps['network private-endpoint update'] = """
+type: command
+short-summary: Update a private endpoint.
+examples:
+  - name: Update a private endpoint.
+    text: az network private-endpoint update -g MyResourceGroup -n MyPE --request-message "test" --tags mytag=hello
+"""
+
+helps['network private-link-service'] = """
+type: group
+short-summary: Manage private link services.
+"""
+
+helps['network private-link-service create'] = """
+type: command
+short-summary: Create a private link service.
+examples:
+  - name: Create a private link service
+    text: az network private-link-service create -g MyResourceGroup -n MyPLSName --vnet-name MyVnetName --subnet MySubnet --lb-name MyLBName --lb-frontend-ip-configs LoadBalancerFrontEnd -l centralus
+"""
+
+helps['network private-link-service delete'] = """
+type: command
+short-summary: Delete a private link service.
+"""
+
+helps['network private-link-service list'] = """
+type: command
+short-summary: List private link services.
+"""
+
+helps['network private-link-service show'] = """
+type: command
+short-summary: Get the details of a private link service.
+"""
+
+helps['network private-link-service update'] = """
+type: command
+short-summary: Update a private link service.
+examples:
+  - name: Update a private link service
+    text: az network private-link-service update -g MyResourceGroup -n MyPLSName --visibility SubId1 SubId2 --auto-approval SubId1 SubId2
+"""
+
+helps['network private-link-service connection'] = """
+type: group
+short-summary: Manage private link service endpoint connections.
+"""
+
+helps['network private-link-service connection delete'] = """
+type: command
+short-summary: Delete a private link service endpoint connection.
+"""
+
+helps['network private-link-service connection update'] = """
+type: command
+short-summary: Update a private link service endpoint connection.
+examples:
+  - name: Update the endpoint connections status of private link service
+    text: az network private-link-service connection update -g MyResourceGroup -n tttt.f072a430-2d82-4470-ab30-d23fcfee58d1 --service-name MyPLSName --connection-status Rejected
```

Comment:
then name seems complex for some reason "tttt.f072a430-2d82-4470-ab30-d23fcfee58d1"